radix-engine-interface/src/model/resource/non_fungible_id_type.rs

```diff
@@ -0,0 +1,13 @@
+use crate::*;
+use sbor::*;
+
+/// Represents type of non-fungible id
+#[derive(
+    Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash, Encode, Decode, LegacyDescribe,
+)]
+pub enum NonFungibleIdType {
```

Comment:
Would `NonFungibleIdKind` be more consistent?

Reply:
Decision reverted. Will keep `NonFungibleIdType` to be consistent with doc.